rudra/deployments/emr_scripts/emr_config.py

```diff
@@ -15,7 +15,6 @@ def __init__(self, name, log_uri, ecosystem, s3_bootstrap_uri, training_repo_url
         self.instances = {
             'KeepJobFlowAliveWhenNoSteps': False,
             'TerminationProtected': False,
-            'Ec2SubnetId': 'subnet-50271f16',
             'Ec2KeyName': 'Zeppelin2Spark',
             'InstanceGroups': []
```

Comment:
Are we planning to send this as an argument if needed or this is not needed at all @abs51295 ?

Reply:
Ideally, this shouldn't be needed. PTAL: https://docs.aws.amazon.com/emr/latest/APIReference/API_Ec2InstanceAttributes.html.

Reply:
Got it 
```
Ec2SubnetId
To launch the cluster in Amazon VPC, set this parameter to the identifier of the Amazon VPC subnet where you want the cluster to launch. If you do not specify this value, the cluster is launched in the normal AWS cloud, outside of a VPC.

Amazon VPC currently does not support cluster compute quadruple extra large (cc1.4xlarge) instances. Thus, you cannot specify the cc1.4xlarge instance type for nodes of a cluster launched in a VPC.

Type: String

Required: No
```
This makes sense